widget/embedded/src/components/TokenList/TokenList.tsx

```diff
@@ -0,0 +1,155 @@
+import type { PropTypes, TokenWithAmount } from './TokenList.types';
+import type { CommonProps } from 'react-window';
+
+import { i18n } from '@lingui/core';
+import {
+  Divider,
+  Image,
+  ListItem,
+  ListItemButton,
+  NotFound,
+  Skeleton,
+  Typography,
+  VirtualizedList,
+} from '@rango-dev/ui';
+import React, { forwardRef, useEffect, useState } from 'react';
+
+import { useMetaStore } from '../../store/meta';
+
+import { filterTokens } from './TokenList.helpers';
+import {
+  BalanceContainer,
+  Content,
+  End,
+  List,
+  Tag,
+  Title,
+} from './TokenList.styles';
+
+const PAGE_SIZE = 20;
+const Fixed_Height = 16;
+export function TokenList(props: PropTypes) {
+  const { list, searchedFor, onChange } = props;
```

Comment:
```suggestion
  const { list, searchedFor = "", onChange } = props;
```

And make the `searchedFor` in `PropTypes` optional.